ztf_viewer/model_fit.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pandas as pd
+import requests
+from ztf_viewer.catalogs.ztf_ref import ztf_ref
+from ztf_viewer.exceptions import NotFound, CatalogUnavailable
+from ztf_viewer.util import ABZPMAG_JY, LN10_04
+
+
+class ModelFit:
+    base_url = "http://host.docker.internal:8000/api/v1"
+    bright_fit = "diffflux_Jy"
+    brighterr_fit = "difffluxerr_Jy"
+
+    def __init__(self):
+        self._api_session = requests.Session()
+
+    def fit(self, df, fit_model, ref_mag_values, dr, ebv):
+        path = "/sncosmo/fit"
+        if not ref_mag_values:
+            oid_ref = {}
+            try:
```

Comment:
I don't really understand this part of the code.
1. What `ref_mag_values` supposed to be? True/False? Or None/dict? If the least, than you never use that dict, and also please change the statement to more explicit: `if ref_mag_values is not None`.
2. If the catalog not available, you `pass` and never use the references. It is not a good behavior, because the result will change without user's consent, and do it implicitly without any notification. Maybe it is better to just fail, than do that
3. You are mutating the input dataframe. It is not a good practice, what if the caller would reuse it? Please do something like `df = df.copy()` if you want to change the values.